sqlmesh/integrations/github/cicd/controller.py

```diff
@@ -444,6 +444,10 @@ def modified_snapshots(self) -> t.Dict[SnapshotId, t.Union[Snapshot, SnapshotTab
     def removed_snapshots(self) -> t.Set[SnapshotId]:
         return set(self.prod_plan_with_gaps.context_diff.removed_snapshots)
 
+    @property
+    def is_branched_from_prod(self) -> bool:
```

Comment:
A bit misleading name. It can be branched from anything, the target what matters here. So isn't this more of `targets_prod_branch`?